### ISSUE TITLE
Wrong colors in comparison with screenshot

### ISSUE THREAD

**Issue by danielleontiev** [dracula/vim#136]
### What happened

Colors in my terminal don't match colors on screenshot from main page

### What I expected to happen

Exactly the same look as on the screenshot

### Screenshot

This is how `javascript` looks in my terminal:

[![enter image description here][1]][1]

Problems: 

1) function name, `var` and `this` keywords are grey but should have different colors as on authors screen.

2) `function` keyword is green but should be cyan

3) Current line number is not highlighted

4) Line numbers color is blue but not grey

5) Other colors are not exactly as on screenshot

Screenshot with `pangloss/vim-javascript` installed:

[![enter image description here][2]][2]

Looks like colors was swaped.

Moreover there is no "current line" coloring.

### Machine Info

- **Vim type (`vim`/`gvim`/`neovim`)**: neovim
- **Vim version**: 0.3.4
- **OS**: Linux compaq 4.15.0-48-generic #51~16.04.1-Ubuntu SMP Fri Apr 5 12:01:12 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
- **Terminal/Terminal Emulator/VTE**: `gnome-terminal`, `tilda`, `xterm`, `tilix`, `st` and `xfce4-terminal`
- **`TERM` environment variable**: `xterm-256color`

[1]: https://i.stack.imgur.com/AaAVK.png
[2]: https://i.stack.imgur.com/9E3SU.png

**Comment by dsifford** [comment]
Since this repo is constantly evolving on a fluid basis, it's not feasible to keep the screenshots updated every time we push updates.

We try our best to follow the [running specification](https://dsifford.github.io/dracula-spec/) as close as possible so that the syntax remains uniform between editors.

Current line coloring is defined, so you might just need to pull down master to your plugin folder. (If you're using vim-plug, just do a `:PlugUpdate`).

TL;DR -- The screenshot is outdated. This repo is correct as it stands (for the most part).

**Comment by Nodman** [comment]
Had the same problem
please take a look at  @dsifford answer https://github.com/dracula/vim/issues/117#issuecomment-449427739
helped me to fix

**Comment by danielleontiev** [comment]
Unfortunately it does not work for me. Colors are still different

**Comment by dsifford** [comment]
See my comment above.
